ui.content/src/main/content/jcr_root/conf/aemdemo/settings/wcm/policies/.content.xml

```diff
@@ -13,7 +13,12 @@
                         jcr:title="Content Form"
                         sling:resourceType="wcm/core/components/policy/policy"
                         components="[/apps/aemdemo/components/form/button,/apps/aemdemo/components/form/hidden,/apps/aemdemo/components/form/options,/apps/aemdemo/components/form/text]">
-                        <jcr:content jcr:primaryType="nt:unstructured"/>
+                        <jcr:content
```

Comment:
(cosmetic) In production project, they will ask you to remove replication and rollout records/timestamps from the commit, This data is pure "technical", does not affect the behavior, and is renewed every other time, so there's no point storing it in GIT repository. UUID is probably automatically assigned by the JCR  as well

Reply:
ok, thanks